src/device.js

```diff
@@ -194,3 +194,31 @@ export function isChrome(ua? : string = getUserAgent()) : boolean {
 export function isSafari(ua? : string = getUserAgent()) : boolean {
     return (/Safari/).test(ua) && !isChrome(ua);
 }
+
+export function isSupportedNativeBrowser() : boolean {
```

Comment:
I'm thinking we should keep this one in paypal repo. Since it's extremely paypal specific; other apps may have different definitions of a 'supported native browser'.

Reply:
Great point!